mesonbuild/mdist.py

```diff
@@ -33,6 +33,10 @@
                      'zip': '.zip'}
 
 def add_arguments(parser):
+    '''
+      here we are adding arguments that can be used with
```

Comment:
You don't need to start each comment with "here we are", because it's redundant.

Reply:
Problem fixed.